voice_text fails to build after 2.1.7 was released

The last successful builds appear to be 2.1.6 on all platforms. Though the two releases since then do not mention any changes to that package: 

Kinetic releases:
- https://github.com/ros/rosdistro/pull/17384
- https://github.com/ros/rosdistro/pull/17483

Indigo:
- https://github.com/ros/rosdistro/pull/17484


Here's a sampling of the failing jobs:
http://build.ros.org/view/Ibin_uT64/job/Ibin_uT64__voice_text__ubuntu_trusty_amd64__binary/
http://build.ros.org/view/Ibin_arm_uThf/job/Ibin_arm_uThf__voice_text__ubuntu_trusty_armhf__binary/
http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__voice_text__ubuntu_xenial_amd64__binary/

It appears  to be a missing include:
http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__voice_text__ubuntu_xenial_amd64__binary/40/console
```
00:05:20.293 [ 83%] Building CXX object CMakeFiles/voice_text.dir/src/voice_text.cpp.o
00:05:20.294 /usr/lib/ccache/x86_64-linux-gnu-g++   -DROSCONSOLE_BACKEND_LOG4CXX -DROS_PACKAGE_NAME=\"voice_text\" -I/tmp/binarydeb/ros-kinetic-voice-text-2.1.8/obj-x86_64-linux-gnu/devel/include -I/opt/ros/kinetic/include -I/opt/ros/kinetic/share/xmlrpcpp/cmake/../../../include/xmlrpcpp  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -DNDEBUG -Wdate-time -D_FORTIFY_SOURCE=2    -DUSE_DUMMY_INCLUDE -o CMakeFiles/voice_text.dir/src/voice_text.cpp.o -c /tmp/binarydeb/ros-kinetic-voice-text-2.1.8/src/voice_text.cpp
00:05:22.383 /tmp/binarydeb/ros-kinetic-voice-text-2.1.8/src/voice_text.cpp:17:40: fatal error: voice_text/VoiceTextConfig.h: No such file or directory
00:05:22.383 compilation terminated.
00:05:22.390 CMakeFiles/voice_text.dir/build.make:65: recipe for target 'CMakeFiles/voice_text.dir/src/voice_text.cpp.o' failed
00:05:22.390 make[4]: *** [CMakeFiles/voice_text.dir/src/voice_text.cpp.o] Error 1
00:05:22.390 make[4]: Leaving directory '/tmp/binarydeb/ros-kinetic-voice-text-2.1.8/obj-x86_64-linux-gnu'
00:05:22.390 make[3]: *** [CMakeFiles/voice_text.dir/all] Error 2
00:05:22.390 CMakeFiles/Makefile2:1204: recipe for target 'CMakeFiles/voice_text.dir/all' failed
00:05:22.390 make[3]: Leaving directory '/tmp/binarydeb/ros-kinetic-voice-text-2.1.8/obj-x86_64-linux-gnu'
00:05:22.390 make[2]: *** [all] Error 2
00:05:22.390 Makefile:130: recipe for target 'all' failed
00:05:22.391 make[2]: Leaving directory '/tmp/binarydeb/ros-kinetic-voice-text-2.1.8/obj-x86_64-linux-gnu'
00:05:22.391 dh_auto_build: make -j1 returned exit code 2
00:05:22.391 	cd /tmp/binarydeb/ros-kinetic-voice-text-2.1.8
00:05:22.392 debian/rules:35: recipe for target 'override_dh_auto_build' failed
00:05:22.392 make[1]: Leaving directory '/tmp/binarydeb/ros-kinetic-voice-text-2.1.8'
00:05:22.392 make[1]: *** [override_dh_auto_build] Error 2
00:05:22.394 make: *** [build] Error 2
00:05:22.394 debian/rules:22: recipe for target 'build' failed
00:05:22.394 dpkg-buildpackage: error: debian/rules build gave error exit status 2
00:05:22.396 E: Building failed
00:05:22.401 Traceback (most recent call last):
00:05:22.401   File "/tmp/ros_buildfarm/ros_buildfarm/binarydeb_job.py", line 138, in build_binarydeb
00:05:22.401     subprocess.check_call(cmd, cwd=source_dir)
00:05:22.402   File "/usr/lib/python3.5/subprocess.py", line 581, in check_call
00:05:22.402     raise CalledProcessError(retcode, cmd)
00:05:22.402 subprocess.CalledProcessError: Command '['apt-src', 'build', 'ros-kinetic-voice-text']' returned non-zero exit status 1
00:05:22.402 # END SUBSECTION
```

--- a/3rdparty/voice_text/CMakeLists.txt
+++ b/3rdparty/voice_text/CMakeLists.txt
@@ -32,7 +32,7 @@ set(VT_LIB_PATH /usr/vt/sayaka/M16/bin/${VT_LIB_ARCH}RAMIO/libvt_jpn.so)
     ${catkin_INCLUDE_DIRS}
   )
   add_executable(voice_text src/voice_text.cpp)
-  add_dependencies(voice_text ${PROJECT_NAME}_generate_messages_cpp)
+  add_dependencies(voice_text ${PROJECT_NAME}_generate_messages_cpp ${PROJECT_NAME}_gencfg)
   set_target_properties(voice_text PROPERTIES COMPILE_FLAGS -D_REENTRANT)
 
 if(NOT EXISTS ${VT_LIB_PATH})

